cookbooks/aws-parallelcluster-environment/recipes/config/efs.rb

```diff
@@ -18,7 +18,7 @@
 
 # Identify the previously mounted filesystems and remove them from the set of filesystems to mount
 shared_dir_array.each_with_index do |dir, index|
-  next unless node['cluster']['internal_shared_dirs'].include?(dir) || dir == "/home" || dir == "home"
+  next unless node['cluster']['internal_shared_dirs'].include?(dir) || dir == "/home" || dir == "home" || dir == node['cluster']['internal_initial_shared_dir']
```

Comment:
Just for my better understanding: what's the difference between the directories in `internal_shared_dirs` and the directory `internal_initial_shared_dir`?

Reply:
Clarified offline